lua/utils/maps_helper.lua

```diff
@@ -3,7 +3,7 @@ local map = vim.keymap.set
 
 M.ver = vim.version()
 
-M.init_inbuilt_cscope = function()
+M.init_inbuilt_cscope = function(db_file)
```

Comment:
better to move this function to `cscope/init.lua` since it is not guaranteed that `M.opts.cscope` will be populated in this file.